### PR TITLE
Adds issue templates

### PR DESCRIPTION
**Pull request by neuromusic**
This PR adds 3 issue templates:
- For users to submit bug reports
- For users to submit feature requests
- For the team to submit user stories

### CODE REVIEW

**Comment by shanaxel42** [comment]
Do these templates get automatically rendered when someone makes an issue? How does git know which one to use?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<!-- Please paste or specifically describe the actual output or traceback. -->
+
+#### Versions
+<!--
+import platform; print(platform.platform())
+import sys; print("Python", sys.version)
+import numpy; print("NumPy", numpy.__version__)
+import skimage; print("Scikit-Image", skimage.__version__)
+import starfish; print("starfish", starfish.__version__)
+-->
```
Comment (neuromusic) [inline .github/ISSUE_TEMPLATE/bug_report.md]: are there any other

**Comment by neuromusic** [comment]
@shanaxel42 with just one template, it gets automatically populated. I'm not 100% sure what the interface looks like for multiple templates.

More info: https://help.github.com/en/articles/about-issue-and-pull-request-templates

**Comment by neuromusic** [comment]
that link shows something like this...
image

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ttung) [reply .github/ISSUE_TEMPLATE/bug_report.md]: https://github.com/spacetx/starfish/blob/master/starfish/core/types/_constants.py#L37

Comment (neuromusic) [reply .github/ISSUE_TEMPLATE/bug_report.md]:
```
(starfish) czirwc1osx1002:~ justinkiggins$ ipython
Python 3.6.8 |Anaconda, Inc.| (default, Dec 29 2018, 19:04:46) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]:  
   ...: import platform; print(platform.platform()) 
   ...: import sys; print("Python", sys.version) 
   ...: import numpy; print("NumPy", numpy.__version__) 
   ...: import scipy; print("SciPy", scipy.__version__) 
   ...: import skimage; print("scikit-image", skimage.__version__) 
   ...: import pandas; print("pandas", pandas.__version__) 
   ...: import sklearn; print("sklearn", sklearn.__version__) 
   ...: import xarray; print("xarray", xarray.__version__) 
   ...: import sympy; print("sympy", sympy.__version__) 
   ...: import starfish; print("starfish", starfish.__version__)                
Darwin-18.6.0-x86_64-i386-64bit
Python 3.6.8 |Anaconda, Inc.| (default, Dec 29 2018, 19:04:46) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]
NumPy 1.15.4
SciPy 1.2.0
scikit-image 0.14.1
pandas 0.23.4
sklearn 0.20.2
xarray 0.11.2
sympy 1.3
starfish 0.1.1
```

**Pull request merged**